src/commands/call.ts

```diff
@@ -58,20 +58,7 @@ export class CallCommand extends ApifyCommand<typeof CallCommand> {
         const userInfo = await getLocalUserInfo();
         const usernameOrId = userInfo.username || userInfo.id;
 
-        const forceActorId = this.args.actorId;
-        let actorId;
-        if (forceActorId) {
-            const actor = await apifyClient.actor(forceActorId).get();
-            if (!actor) throw new Error(`Cannot find Actor with ID '${forceActorId}' in your account.`);
-            actorId = actor.username ? `${actor.username}/${actor.name}` : actor.id;
-        } else {
-            actorId = `${usernameOrId}/${localConfig.name}`;
-            const actor = await apifyClient.actor(actorId).get();
-            if (!actor) {
-                throw new Error(`Cannot find Actor with ID '${actorId}' `
-                    + 'in your account. Call "apify push" to push this Actor to Apify platform.');
-            }
-        }
+        const { id: actorId, userFriendlyId } = await this.resolveActorId(apifyClient, localConfig.name as string | undefined, usernameOrId!);
```

Comment:
The not-undefined assertion here spells out trouble - could you rather check it? Or if there's some reasoning why it must be defined, please add a comment.

Reply:
It's used because the type specifies that the value may be missing, but we also ensure the user is logged in a few lines higher when we get the apify client (in which case either username or id is guaranteed to be set, and if not...people shouldn't edit those internal files)

Reply:
Eeeh, people shouldn't edit internal files, but... things happen. I'd recommend being defensive and throwing an Error.